src/cache/cache_las.c

```diff
@@ -1489,12 +1504,78 @@ __wt_find_lookaside_upd(
 		if (upd_type == WT_UPDATE_BIRTHMARK && ignore_birthmark)
 			break;
 
-		/* It is okay to locate an older update in lookaside,
-		 * but we proceed only if we find a matching record. */
-		WT_ASSERT(session, start_ts >= las_timestamp);
+		/*
```

Comment:
I suggest removing this comment. I do not think it is relevant after removing the assert

Reply:
067612a